src/pages/index.tsx

```diff
@@ -155,10 +154,23 @@ const BlockList: React.FC<BlockListProps> = ({ blocks }) => {
       data: 'string',
       isPadded: true,
     },
+    {
+      key: 'coulmnMiner',
+      name: 'Miner',
+      fieldName: 'miner',
+      minWidth: 200,
+      maxWidth: 450,
+      isRowHeader: true,
+      isResizable: true,
+      isSorted: false,
+      isSortedDescending: true,
+      data: 'string',
+      isPadded: true,
+    },
     {
       key: 'columnTxNumber',
       name: 'Tx #',
-      fieldName: null,
+      fieldName: undefined,
```

Comment:
is it intended?

Reply:
Yes. `fieldName` is optional, so it can be undefined.
But, now I think it's better to remove the line.